dependencies/pom.xml

```diff
@@ -97,6 +97,10 @@
         <version.lib.junit>5.9.3</version.lib.junit>
         <version.lib.junit-platform-testkit>1.11.3</version.lib.junit-platform-testkit>
         <version.lib.kafka>3.8.1</version.lib.kafka>
+        <!-- Kotlin: dependency convergence requirement, we never use this directly -->
+        <version.lib.kotlin>1.9.10</version.lib.kotlin>
+        <version.lib.langchain4j>0.35.0</version.lib.langchain4j>
```

Comment:
Looks like 0.36.2 is the latest (released) version.

Reply:
Yes, there is also 1.0-alpha, we will migrate in the next releases.

Reply:
Created #9746 to collect follow ups